alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -94,6 +94,12 @@ class AlertIndices(
         /** The index name pattern to query all alerts, history and current alerts. */
         const val ALL_INDEX_PATTERN = ".opendistro-alerting-alert*"
 
+        /** The alias of the index in which to write alert finding */
+        const val FINDING_INDEX = ".opendistro-alerting-alert-finding"
```

Comment:
Need to copied version of `HISTORY_WRITE_INDEX` and `HISTORY_INDEX_PATTERN` for `FINDINGS`